From 1e1d834ffdd2ba5e42b8b08f61f7b8034858879e Mon Sep 17 00:00:00 2001
From: mglae <mglmail@arcor.de>
Date: Wed, 20 Sep 2017 19:52:57 +0200
Subject: [PATCH] wrapper: add __xstat() and __fxstat() for libdvdnav

---
 xbmc/cores/DllLoader/exports/wrapper.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/xbmc/cores/DllLoader/exports/wrapper.c b/xbmc/cores/DllLoader/exports/wrapper.c
index e3636623ef8f..8783934ac8aa 100644
--- a/xbmc/cores/DllLoader/exports/wrapper.c
+++ b/xbmc/cores/DllLoader/exports/wrapper.c
@@ -410,6 +410,11 @@ int __wrap_stat(const char *path, struct _stat *buffer)
   return dll_stat(path, buffer);
 }
 
+int __wrap___xstat(int __ver, const char *__filename, struct stat *__stat_buf)
+{
+  return dll_stat(__filename, __stat_buf);
+}
+
 int __wrap___xstat64(int __ver, const char *__filename, struct stat64 *__stat_buf)
 {
   return dll_stat64(__filename, __stat_buf);
@@ -440,6 +445,11 @@ int __wrap___fxstat64(int ver, int fd, struct stat64 *buf)
   return dll_fstat64(fd, buf);
 }
 
+int __wrap___fxstat(int ver, int fd, struct stat *buf)
+{
+  return dll_fstat(fd, buf);
+}
+
 int __wrap_fstat(int fd, struct _stat *buf)
 {
   return dll_fstat(fd, buf);
