From 8f1172db5f6ff10f29a41df57f9b1b8ef602e736 Mon Sep 17 00:00:00 2001
From: kszaq <kszaquitto@gmail.com>
Date: Sat, 15 Apr 2017 21:40:54 +0200
Subject: [PATCH 04/43] VideoSyncAML: Increase rendering thread prioroty

---
 xbmc/windowing/amlogic/VideoSyncAML.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/xbmc/windowing/amlogic/VideoSyncAML.cpp b/xbmc/windowing/amlogic/VideoSyncAML.cpp
index d2a9f98..0916354 100644
--- a/xbmc/windowing/amlogic/VideoSyncAML.cpp
+++ b/xbmc/windowing/amlogic/VideoSyncAML.cpp
@@ -52,6 +52,9 @@ void CVideoSyncAML::Run(CEvent& stopEvent)
   /* This shouldn't be very busy and timing is important so increase priority */
   CThread::GetCurrentThread()->SetPriority(CThread::GetCurrentThread()->GetPriority() + 1);
 
+  /* This shouldn't be very busy and timing is important so increase priority */
+  CThread::GetCurrentThread()->SetPriority(CThread::GetCurrentThread()->GetPriority()+1);
+
   while (!stopEvent.Signaled() && !m_abort)
   {
     int countVSyncs(1);
-- 
2.7.4

