From d3b529e5b2af541df5c4354b7d78e49712d24beb Mon Sep 17 00:00:00 2001
From: boot2k3 <freebox_2002@mail.ru>
Date: Sat, 31 Mar 2018 16:35:21 +0300
Subject: [PATCH 32/43] Revert FlushRenderer in onLostDisplay()

---
 xbmc/cores/VideoPlayer/VideoPlayer.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/xbmc/cores/VideoPlayer/VideoPlayer.cpp b/xbmc/cores/VideoPlayer/VideoPlayer.cpp
index 6b52ebb..066383d 100644
--- a/xbmc/cores/VideoPlayer/VideoPlayer.cpp
+++ b/xbmc/cores/VideoPlayer/VideoPlayer.cpp
@@ -5046,7 +5046,6 @@ void CVideoPlayer::OnLostDisplay()
   m_VideoPlayerVideo->SendMessage(new CDVDMsgBool(CDVDMsg::GENERAL_PAUSE, true), 1);
   m_clock.Pause(true);
   m_displayLost = true;
-  FlushRenderer();
 }
 
 void CVideoPlayer::OnResetDisplay()
-- 
2.7.4

