From cdcc5f414aed9f4084ed102fc5f367e51ae1c8c5 Mon Sep 17 00:00:00 2001
From: cdu13a <cdu13a@gmail.com>
Date: Tue, 17 Sep 2019 15:40:09 -0400
Subject: [PATCH 40/43] aml-settings: minor cleanup to the limit 8bit setting

---
 xbmc/windowing/amlogic/WinSystemAmlogic.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/xbmc/windowing/amlogic/WinSystemAmlogic.cpp b/xbmc/windowing/amlogic/WinSystemAmlogic.cpp
index ca4e2d9..e893774 100644
--- a/xbmc/windowing/amlogic/WinSystemAmlogic.cpp
+++ b/xbmc/windowing/amlogic/WinSystemAmlogic.cpp
@@ -106,6 +106,10 @@ bool CWinSystemAmlogic::InitWindowSystem()
      CLog::Log(LOGDEBUG, "CWinSystemAmlogic::InitWindowSystem -- Limiting display to 8bit colour depth");
      if (attr.find("10bit") != std::string::npos)
        attr.replace(attr.find("10bit"),5,"8bit");
+     else if (attr.find("12bit") != std::string::npos)
+       attr.replace(attr.find("12bit"),5,"8bit");
+     else if (attr.find("8bit") != std::string::npos)
+       ; //do nothing
      else
        attr.append("8bit");
   }
-- 
2.7.4

